src/Solver.cpp

```diff
@@ -53,9 +53,6 @@ Solver::Solver(TrackGenerator* track_generator) {
  */
 Solver::~Solver() {
 
-  if (_FSR_volumes != NULL)
```

Comment:
Does the `Solver` never actually use the FSR volumes? Or does it only retrieve the array from the `TrackGenerator` into temporary variables now?